Njord/src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -23,15 +38,56 @@ public class ElevatorSubsystem extends SubsystemBase {
   public enum ElevatorState {
     S_GoingUp, S_GoingDown, S_Stopped
     }
+
+  private final MutVoltage mVoltage = Volts.mutable(0);
+  private final MutDistance mDistance = Meters.mutable(0);
+  private final MutLinearVelocity mVelocity = MetersPerSecond.mutable(0);
+
+private Voltage setVolts = (Voltage volts) -> setVoltage(volts);
+private double setLog = (SysIdRoutineLog) -> setLog(SysIdRoutineLog);
+
+
+
+private final SysIdRoutine mSysIdRoutine = 
+   new SysIdRoutine(
+     new SysIdRoutine.Config(Volts.per(Seconds).of(1.2), Volts.of(12), Seconds.of(10)),
```

Comment:
This example is from the docs where the default values are overriden. Let's use the default values to start with. The default values seems sane.

```suggestion
     new SysIdRoutine.Config(),
```